sigopt/hyperopt/base.py

```diff
@@ -38,35 +41,32 @@ def upload(self, trials=None):
     self.uploaded_tids.update(ids)
     return ids
 
-  def _upload(self, trials):
-    ids = {}
-    for trial in trials:
-      tid = trial['tid']
-      result = trial['result']
-      metrics = {k:v for k, v in result.items() if isinstance(v, (int, float))}
-      parameters = self.trial_parameters(trial)
-      metadata = {'optimizer': 'hyperopt'}
-      status = result.get('status')
-      run = self.factory.create_run(metadata=metadata)
-      with run:
-        self.log_run_params(run, parameters)
-        if status == STATUS_OK:
-          run.log_metrics(metrics)
-        elif status == STATUS_FAIL:
-          run.log_failure()
-        else:
-          raise ValueError(f'status must be {STATUS_OK} or {STATUS_FAIL}, actully {status}')
-      ids[tid] = run.id
-    return ids
+  def trial_to_run(self, trial):
+    metadata = {'optimizer': 'hyperopt'}
+    result = trial['result']
+    metrics = {k:v for k, v in result.items() if isinstance(v, (int, float))}
+    parameters = self.trial_parameters(trial)
+    status = result.get('status')
+    run = DictRunContext(name=get_default_name(self.factory.project), metadata=metadata)
+    run.log_parameters(parameters,
+                       source=HYPEROPT_SOURCE_NAME,
+                       source_meta={
+                         'sort': HYPEROPT_SOURCE_PRIORITY,
+                         'default_show': True
+                       })
```

Comment:
we typically use the following indentation for something like that:
```
  run.log_parameters(
     parameters,
     source=HYPEROPT_SOURCE_NAME,
     ...
  )
```

Reply:
Fixed.